scraper.rb

```diff
@@ -18,15 +18,17 @@ def scrape(h)
 
 def scrape_list(list_url)
   page = scrape(list_url => MembersPage)
-  page.member_urls.each { |url| scrape_person(url) }
+  page.member_urls.each { |url| scrape_person(url, term: page.term) }
   scrape_list(page.next_page) unless page.next_page.to_s.empty?
 end
 
-def scrape_person(url)
-  data = scrape(url => MemberPage).to_h
+def scrape_person(url, term:)
```

Comment:
it's a little bit clumsy to need to pass the 'term' around here, when the only reason it's needed is for combining with the data that this method is gathering. The method is already doing two things, so this is largely an inherited problem, but I think this is the time to rectify that, rather than pushing it even further in the wrong direction. So I think it would be much cleaner to take this opportunity to turn this into a `person_data` method which simply returns the scraped data for the person, and hoist the `save_sqlite` out of this method.

(The ideal would be to split the `save_sqlite` out entirely from the data-gathering, so there's just a single delete and re-save after all the scraping, thus ensuring the data doesn't end up in a half-baked state if something fails half-way through, as per current conventions, but that's not quite as necessary here.)